### PR TITLE
{bio}[system/system] MiXCR v4.6.0 w/ Java 17

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/m/MiXCR/MiXCR-4.6.0-Java-17.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+checksums = ['05db1276951a2e656d0a7bf4e2b1fff326733a5f961a9d4829be139852fabe13']
+
+dependencies = [('Java', '17')]
+
+sanity_check_paths = {
+    'files': ['mixcr', 'mixcr.jar'],
+    'dirs': [],
+}
+
+sanity_check_commands = ["command -v mixcr"]
```
Comment (smoors) [inline easybuild/easyconfigs/m/MiXCR/MiXCR-4.6.0-Java-17.eb:24]: @pavelToman it's better to run `mixcr` directly. the following command does not need a license:
```suggestion
sanity_check_commands = ["mixcr -v"]
```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3159.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/33bad4934c319555b5714493e9367081 for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @pavelToman!

**Pull request merged**